CONTRIBUTING.md

```diff
@@ -17,26 +17,31 @@
     ```
 
 4. Have a look around to get a feel for the app.
-5. Run the tests. We only take pull requests with passing tests, and it's great to confirm that you have a clean slate.
+
+5. (Optional) Note that to be able to use the page as an admin, you must first give yourself admin privileges. Make sure you have started your app and signed up as an user on your locally running app.
+  * Non-docker users can run this on command line: `rails runner "Member.find_by(name: '<your first name>').add_role(:admin)"`
```

Comment:
@turger this is a minor point but can we suggest people find themselves by email address? I think that's a more reliable piece of information then first name.

Reply:
Very good point, I'll change name to email!